handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,57 @@
+# MinIO as storage provider for the FileService
+
+SystemLink FileService allows you to configure and use the Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the FileService also works with MinIO, a server-side software storage stack that is compatible with Amazon S3.
+
+To use MinIO with SystemLink, you need SystemLink version 2020 R3 or later.
+
+To use MinIO as a storage provider, set up the MinIO server and configure the File Service to use that server.
+
+## Running MinIO server
+
+1. In order to run the MinIO server, download the server from the [MinIO website](https://min.io/download). From the command line, run the following command to run the server:
+
+    ```bash
+    minio server <PathToRootFolder>
+    ```
+
+2. After starting the MinIO server, connect to the endpoint that is printed out in the command line by using a web browser and create a bucket by using the `+` button in the right bottom corner. This is only required if you run the MinIO server for the first time.
+
+    !!! note "Note"
+        - It is recommended to set non-default access key and secret key. You can do that by setting the environment variables `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`.
+        - There's no bucket region necessary for using MinIO, but you have to apply a bucket region for the FileService to work. Just use `us-east-1`.
```

Comment:
This note seems more important than the other 2 -- like it might prevent a user from completing the task successfully if they do not apply a bucket region. If this is the case, configuring this setting should be its own step.